x-pack/platform/plugins/shared/onechat/public/application/routes.tsx

```diff
@@ -15,6 +15,11 @@ import { OnechatToolCreatePage } from './pages/tool_create';
 import { OnechatToolDetailsPage } from './pages/tool_details';
 import { OnechatToolsPage } from './pages/tools';
 
+/*
+  Note: each route rendered here should be wrapped with PageWrapper
+  This allows each route to set the correct breadcrumbs before checking access
+*/
```

Comment:
So why do we need that compared to using the HOC on top of the router component a single time?